yt/utilities/cosmology.py

```diff
@@ -319,9 +323,9 @@ def lookback_time(self, z_i, z_f):
         return (trapzint(self.age_integrand, z_i, z_f) / \
                 self.hubble_constant).in_base(self.unit_system)
     
-    def hubble_time(self, z, z_inf=1e6):
```

Comment:
I realize we talked about this already, but would you be ok with keeping the old calculation but deprecating the name `hubble_time`. You could then add a new function which returns one over the hubble constant. I worry about users who will now get different results from this function in the next yt version with no warning. If you feel that's not a concern worth worrying about I'd love to hear why you think so.

Reply:
Sure, I can do that. I think the counterargument is that this could be seen as constituting a bugfix, but since the correct behavior is basically trivial, I can just include that in the deprecation warning.